go/kbfs/libkbfs/folder_block_manager.go

```diff
@@ -1376,6 +1376,7 @@ func (fbm *folderBlockManager) doChunkedGetNonLiveBlocks(
 
 func (fbm *folderBlockManager) doCleanDiskCache(cacheType DiskBlockCacheType) (
 	err error) {
+	fbm.log.CWarningf(context.Background(), "Starting to clean %s", cacheType)
```

Comment:
We should get rid of this too -- I intentionally didn't have a message until after we figured out if cleaning was needed, to avoid log pollution.